### DISCUSSION TITLE
next page failed

### DISCUSSION THREAD

**Discussion by as93717913** [tchiotludo/akhq#1416] [Q&A]
I click the next page button in the topic tab it will show no data available,
![image](https://user-images.githubusercontent.com/16937091/206402782-25bda432-be58-44d1-a037-a73426ae6c91.png)
but if I enter same url(with the query string), the result will show correctly.
![image](https://user-images.githubusercontent.com/16937091/206402976-2d518853-245a-49aa-8808-bf87fb85ef60.png)

**Comment by tchiotludo** [comment]
Is there an timeout delay that can prevent the loading of message? 
See [here](https://akhq.io/docs/configuration/akhq.html): try increase this `akhq.topic-data.poll-timeout`
